app/src/pages/authoring/[resourceType]/index.tsx

```diff
@@ -1,10 +1,36 @@
-import { Center, Text } from '@mantine/core';
+import { trpc } from '@/util/trpc';
+import { Center, Text, Divider } from '@mantine/core';
+import { useRouter } from 'next/router';
+import { ArtifactResourceType, ResourceInfo } from '@/util/types/fhir';
+import ResourceCards from '@/components/ResourceCards';
+import { extractResourceInfo } from '@/util/resourceCardUtils';
+import { useMemo } from 'react';
+
 export default function ResourceAuthoringPage() {
+  const router = useRouter();
+  const { resourceType } = router.query;
+
+  const { data: artifacts } = trpc.draft.getDrafts.useQuery(resourceType as ArtifactResourceType);
+
+  const resourceCardContent: ResourceInfo[] = useMemo(() => {
+    return (artifacts ?? []).map(a => extractResourceInfo(a));
+  }, [artifacts]);
+
   return (
     <div>
       <Center>
-        <Text c="gray">Coming Soon</Text>
+        <Text c="gray" fz="xl">
+          Available Draft {resourceType} Resources
+        </Text>
       </Center>
+      <Divider my="md" />
```

Comment:
Not sure if this is a matter of importance or not, but I noticed on both my smaller/bigger monitor that the` "Available Draft {resourceType} Resouces" header` along with the cards seem to be slightly off-centered from the `"Repository/Authoring" tabs` in the navigation bar at the top. This might not matter if we are planning on adding more tabs to the navigation bar, but maybe we could move the header and the cards so that they are centered underneath those two tabs? 

Here is what the image looks like on my screen:
![UncenteredHeader](https://github.com/projecttacoma/measure-repository/assets/98363677/94f74313-4afb-41ea-af3e-8e60b34d0b39)

Reply:
I think the reason why we are seeing this is because the "Repository"/"Authoring" tabs are centered in respect to the entire screen, whereas the "Available Draft Library Resources" text is centered in respect to the `index.tsx` page, which is separate from the sidebar that we see on the left. If we were to change the centering here, we would need to do the same on a few other pages (like the search results, which you mentioned). Since the precedent was set to have the text centered in respect to the page rather than the entire screen, I think I will keep this as-is for now and we can consider making a task to update the styling across all the necessary pages

Reply:
This one's tricky because the tabs are centered within the top bar, whereas the header is centered within its resource list section (which is offset from the top bar by the navigation on the left). Because the tabs also change the navigation, they should remain within the top bar. I don't mind the different centeredness too much, but if others find it jarring, the tabs could potentially be moved to be left aligned so that the different centeredness isn't a bother. Next to the main Measure Repository label/link, something like this:
<img width="374" alt="Screen Shot 2023-06-20 at 12 03 14 PM" src="https://github.com/projecttacoma/measure-repository/assets/2643955/c4a0d126-bd82-46a1-8232-b18b34560b0e">

Reply:
Honestly, I prefer it centered like it was, but I don't have great reason for that other than it's my gut feel :)

Reply:
Sounds good to me! This is really more of a subjective aspect of the UI so it's more up to personal preference than anything :)